src/client/src/pages/settings/admin/workgroupAdministration.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { GridEventListener } from "@mui/x-data-grid";
+import { User, Workgroup } from "../../../api/apiInterfaces.ts";
+import { fetchWorkgroups } from "../../../api/workgroup.ts";
+import { useApiRequest } from "../../../hooks/useApiRequest.ts";
+import { WorkgroupTable } from "./workgroupTable.tsx";
+
+interface WorkgroupAdministrationProps {
+  users: User[];
+}
+
+export const WorkgroupAdministration: FC<WorkgroupAdministrationProps> = ({ users }) => {
+  const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(true);
+  const [workgroups, setWorkgroups] = useState<Workgroup[]>();
+  const { callApiWithErrorHandling } = useApiRequest();
+
+  useEffect(() => {
+    setIsLoading(true);
+    const getWorkgroups = async () => {
+      const workgroups: Workgroup[] = await callApiWithErrorHandling(fetchWorkgroups, []);
+      setWorkgroups(workgroups);
+      setIsLoading(false);
```

Comment:
REQ: `setIsLoading(false)` wird unter Umständen, z.B. bei einem unerwarteten Fehler, nicht aufgerufen. Das sollte entsprechend in ein `finally`-Block verschoben werden, mit Error-Handling.

Reply:
Habe den isLoading state entfernt, war mir eh ein Dorn im Auge den noch separat managen zu müssen. Ich denke es geht auch ohne.